freqtrade/rpc/telegram.py

```diff
@@ -215,8 +215,8 @@ def _status(self, update: Update, context: CallbackContext) -> None:
                     "*Open Rate:* `{open_rate:.8f}`",
                     "*Close Rate:* `{close_rate}`" if r['close_rate'] else "",
                     "*Current Rate:* `{current_rate:.8f}`",
-                    "*Close Profit:* `{close_profit}`" if r['close_profit'] else "",
-                    "*Current Profit:* `{current_profit:.2f}%`",
+                    "*Close Profit:* `{close_profit}`" if r['close_profit_perc'] else "",
```

Comment:
-- is not it by mistake?
(close_profit is printed and existence of close_profit_perc is checked)?

also, it will print "" if close_profit_perc is 0...